app/uk/gov/hmrc/economiccrimelevyreturns/views/AlreadySubmittedReturnView.scala.html

```diff
@@ -30,9 +30,9 @@
     <h1 class="govuk-heading-l">@messages("alreadySubmittedReturn.heading", fyStartYear, fyEndYear)</h1>
 
     <p class="govuk-body">@messages("alreadySubmittedReturn.p1", fyStartYear, fyEndYear, submittedDate)</p>
-    @if(!appConfig.privateBetaEnabled){
-        <p class="govuk-body">@Html(messages("alreadySubmittedReturn.p2", "#TODO"))</p>
-    }
+@*    @if(appConfig.amendReturnsEnabled){*@
```

Comment:
Are these lines meant to be commented out?

Reply:
Yes, as it's set to a TODO until the amend returns work comes in. I don't know why it was left in, in a TODO state in the first place.